.github/buildomat/jobs/sample.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#:
+#: name = "build-and-test (ubuntu-22.04)"
+#: variety = "basic"
+#: target = "ubuntu-22.04"
+#: rust_toolchain = "nightly-2024-04-04"
```

Comment:
do you know if there's a way for this to be sourced from the `rust-toolchain.toml` file?

if not, it might be worth adding a comment in that file saying "hey, you have to update the Buildomat job definition if you change this". maybe also worth an upstream Buildomat feature request for `rust-toolchain.toml` awareness.